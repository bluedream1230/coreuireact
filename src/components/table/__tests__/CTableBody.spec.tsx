import * as React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { CTableBody } from '../../../index'

test('loads and displays CTableBody component', async () => {
  const { container } = render(<CTableBody/>)
  expect(container).toMatchSnapshot()
})

test('CTableBody customize', async () => {
    const { container } = render(
        <CTableBody
            className="bazinga"
            color="info"
        >
            Test
        </CTableBody>
    )
    expect(container).toMatchSnapshot()
    expect(container.firstChild).toHaveClass('table-info')
    expect(container.firstChild).toHaveClass('bazinga')
    expect(container.firstChild).toHaveTextContent('Test')
})