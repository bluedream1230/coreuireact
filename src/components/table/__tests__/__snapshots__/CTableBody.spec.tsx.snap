// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTableBody customize 1`] = `
<div>
  <tbody
    class="table-info bazinga"
  >
    Test
  </tbody>
</div>
`;

exports[`loads and displays CTableBody component 1`] = `
<div>
  <tbody />
</div>
`;
